share/completions/port.fish

```diff
@@ -0,0 +1,115 @@
+complete -f -n '__fish_port_no_subcommand' -c port -a 'rdependents'  --description 'Recursively lists the installed ports that depend on the port portname'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'rdeps'  --description 'Recursively lists the other ports that are required to build and run portname'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'reload'  --description 'A shortcut to launchctl, like load and unload, but reloads the daemon'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'rev-upgrade'  --description 'Manually check for broken binaries and rebuild ports containing broken binaries'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'search'  --description 'Search for a port using keywords'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'select'  --description 'For a given group, selects a version to be the default by creating appropriate symbolic links'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'selfupdate'  --description 'Upgrade MacPorts itself and update the port definition files.'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'setrequested'  --description 'Mark portname as requested'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'setunrequested'  --description 'Mark portname as unrequested'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'sync'  --description 'Update the port definition files'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'test'  --description 'Run test phase of a port'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'unarchive'  --description 'Extract the destroot of the given ports from a binary archive'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'uninstall'  --description 'Remove a previously installed port'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'unload'  --description 'A shortcut to launchctl, like load, but unloads the daemon'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'unsetrequested'  --description 'Mark portname as unrequested'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'upgrade'  --description 'Upgrade a port to the latest version'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'url'  --description 'Displays the URL for the path of the given portname, which can be passed as port-url'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'usage'  --description 'Displays a condensed usage summary'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'variants'  --description 'Print a list of variants with descriptions provided by a port'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'version'  --description 'Print the MacPorts version'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'work'  --description 'Displays the path to the work directory for portname'
+
+complete -c apt-get -s v --description 'Verbose mode, generates verbose messages'
```

Comment:
Should these apt-get entries be here?

Reply:
The options and descriptions are correct, but the command should be port, not apt-get. I used apt-get completions as a template, seems to be a relic from that.